grizzly_extras/async_message/mq/__init__.py

```diff
@@ -260,7 +261,7 @@ def _request(self, request: AsyncMessageRequest) -> AsyncMessageResponse:
                     payload = request.get('payload', None)
                     if self.header_type:
```

Comment:
could remove one branch by doing `if self.header and self.header == 'rfh2':`.

Reply:
check!